layouts/404.html

```diff
@@ -3,11 +3,11 @@
         <div class="content">
             <div id="intro-main">
                 <div id="site-logo">
-                    <img src="/images/irl-logo.png" srcset="/images/irl-logo-high-res.png 2x" alt="">
+                    <img src="/images/irl-logo.png" srcset="/images/irl-logo-high-res.png 2x" alt="IRL">
                 </div>
 
                 <div id="site-title">
-                    <h1>IRL: Because Online Life Is Real Life</h1>
+                    <h1>Because Online Life Is Real Life</h1>
```

Comment:
Totally nitpicking, but... I feel like removing "IRL" from the title text changes the semantics, and now the `h1` is more of a tagline and the real site title is just the alt text on the image above, which has no special semantics since it's just a `div`. We could put the image inside the `h1` but that will take a little more effort to adjust layout and styling.

Or else just make `#site-logo` an `h1` instead of a `div`, and make this an `h2`? Or even just a `p` (if it's only a tagline and an `h2` would change the whole outline).

Reply:
Other pages do it this way already, I was just trying to fix what is essentially a typo here.

Reply:
Yeah, I think those could be improved in the same way. Just a thought, it doesn't have to be part of this PR.